ci/prow/cluster.yaml

```diff
@@ -148,7 +148,7 @@ spec:
       serviceAccountName: "plank"
       containers:
       - name: plank
-        image: gcr.io/k8s-prow/plank:v20190109-716fe27
+        image: gcr.io/k8s-prow/plank:v20190311-a967141
```

Comment:
Please add a comment here and in the generator about keeping the tags in sync, so this doesn't happen again. Also thanks for switching to tags instead of SHA for the images, so we can have such parity.

Reply:
good idea, done

Reply:
Please add a comment here as well.

Reply:
done